langchain/agents/tests/agent.test.ts

```diff
@@ -3,10 +3,11 @@ import { OpenAI } from "../../llms/openai";
 import { loadAgent } from "../load";
 import { AgentExecutor, Tool } from "../index";
 import { SerpAPI } from "../tools/serpapi";
+import { Calculator } from "../tools/calculator";
 
 test("Run agent from hub", async () => {
   const model = new OpenAI({});
-  const tools: Tool[] = [SerpAPI()];
+  const tools: Tool[] = [SerpAPI(), Calculator];
```

Comment:
Wasn't sure about this one? Tool interface seems to be an object, but the SerpAPI is implemented as a function that returns that object. Do we want them all to be "functions returning objects"? In which case maybe Tool should just be a regular class?

Reply:
I'll do that in a separate PR, and then we can decide

Reply:
sounds awesome

Reply:
https://github.com/hwchase17/langchainjs/pull/9